build/constants.yaml

```diff
@@ -320,7 +320,7 @@
       file-name-prefix: __
 - name: java-versions
   constants:
-    java-version: 17.0.2
+    java-version: 11.0.14
```

Comment:
I understand that we're adding support for this older version, but could you please clarify why we're setting it as the java-version here?

Reply:
Just to clarify, we will be needing both the Java versions (17.0.2 and 11.0.14)

Reply:
17.0.2 is already in our storage account. Changes to this file will not remove any SDKs. Since `11.0.14` is an older version, I was considering in using this PR to trigger tests for validation and deployment, then closing the PR after the binaries have been released

Reply:
Thanks for clarifying. Even if we revert some of these changes, I imagine we'll still keep the versionsToBuild.txt and JavaDynamicInstallationTest.cs changes, right?

Reply:
In `build/__javaVersions.sh` can we then have two constants? One for 17 and another for 11? 
It will be easier to use in `vso.focal.dockerfile` which I am looking forward to modify soon.

Reply:
or I can add that variable myself when I create the PR. Anything which is easier for you would work.

Reply:
I've added a `JAVA11_VERSION`. Thanks for reviewing :)